scb-engine/src/main/java/io/securecodebox/engine/rest/SecurityTestDefinitionResource.java

```diff
@@ -59,6 +70,12 @@
     })
     @RequestMapping(method = RequestMethod.GET)
     public ResponseEntity<List<String>> getSecurityTestDefinitions(){
+        try {
+            authService.isAuthorizedFor(ResourceType.SECURITY_TEST_DEFINITION, PermissionType.READ);
+        }catch (InsufficientAuthenticationException e){
+            return ResponseEntity.status(401).build();
```

Comment:
If the user is not authenticated, he should not be able to call this mehtod (because the authentication filter (Class BasicAuthWebSecurityConfig) should hopefully prohibits this).

I think a 403 (Forbidden) would be the correct status code